decidim-accountability/config/i18n-tasks.yml

```diff
@@ -0,0 +1,10 @@
+base_locale: en
+locales: [en, es, ca]
+ignore_unused:
+- "decidim.features.accountability.name"
+- "decidim.features.accountability.settings.*"
+- "decidim.resource_links.*"
+- "decidim.accountability.models.project.valid_statuses.*"
+- activerecord.attributes.*
+- activemodel.attributes.*
+- activemodel.errors.*
```

Comment:
We only have a single `i18n-tasks` config file now, this config should be merged there.